test/app.hooks.test.js

```diff
@@ -17,6 +17,47 @@ describe('Sanitization Hook', () => {
 		expect(result.data).to.not.be.an('undefined');
 		expect(result.data.testString).to.equal(testString);
 	});
+
+	it('hook sanitizes when "before" type is triggered by checking the existence of "data" property', () => {
+		const testString = '<script>alert("test");</script><h1>test</h1>';
+		const sanitizedTestString = 'test';
+		const context = {
+			path: 'not_authentication',
+			data: { testString },
+			type: 'before',
+		};
+		const result = sanitizeDataHook(context);
+		expect(result.data).to.not.be.an('undefined');
+		expect(result.data.testString).to.equal(sanitizedTestString);
+		expect(result.result).to.be.an('undefined');
+
+	});
+
+	it('hook sanitizes when it is triggered in "after" type by checking the existence of "result" property', () => {
+		const testString = '<script>alert("test");</script><h1>test</h1>';
+		const sanitizedTestString = 'test';
+		const context = {
+			path: 'not_authentication',
+			result: { testString },
+			type: 'after',
+		};
+		const result = sanitizeDataHook(context);
+		expect(result.result).to.not.be.an('undefined');
+		expect(result.result.testString).to.equal(sanitizedTestString);
+		expect(result.data).to.be.an('undefined');
+
+	});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Block must not be padded by blank lines.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5471106)